setup.py

```diff
@@ -1,38 +1,32 @@
 import codecs
 import os
-import re
 import sys
 
 from setuptools import find_packages, setup
 
-here = os.path.abspath(os.path.dirname(__file__))
 
-
-def read(*parts):
+def codopen(rel_path):
```

Comment:
I would keep the `read` function the same and use `splitlines()` in `get_version` to get each line instead.